crates/e2e/tests/e2e/eth_integration.rs

```diff
@@ -75,28 +74,6 @@ async fn eth_integration(web3: Web3) {
     let http_factory = HttpClientFactory::default();
     let client = http_factory.create();
 
-    // Test fee endpoint
```

Comment:
Should we test `/quote` endpoint here instead?

Reply:
I don't think it's really needed because this test is just checking for the status code and we already create a quote when handling a new order. But I could add it back.

Reply:
If we already "use" the `/quote` path elsewhere, I'm OK with removing this test here.

Reply:
I mean internally we create a quote but not explicitly in this test.

Reply:
Added a test back